app/boards/shields/kyria/boards/nrfmicro_11_flipped.overlay

```diff
@@ -0,0 +1,28 @@
+&spi1 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
+	mosi-pin = <6>;
+	// Unused pins, needed for SPI definition, but not used by the ws2812 driver itself.
+	sck-pin = <5>;
+	miso-pin = <7>;
```

Comment:
Pins 5 and 7 seem to be used on certain parts of revisions 1.2 and 1.3: https://github.com/joric/nrfmicro/wiki/Pinout#nrfmicro-pinout

Should look into other unused pins, so that others don't copy these two pins in when they inevitably use this for their configs.